### ISSUE TITLE
Elevated circulating sebacic acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6426]
**Preferred term label:** Elevated circulating sebacic acid concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of sebacic acid in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Dicarboxylic acidemia HP:0040145

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
